### PR TITLE
Add production content

### PR DESCRIPTION
**Pull request by rogeruiz**
This patch series adds all of the content from the production site `https://vote.usa.gov/`.

This work was done mostly mechanically, so it's a direct port of the links and associated `registration_types` ( for example: `by-mail`, `online`, `in-person`, `not-needed` ) from the production site. I've verified all the states and territories render correctly and that the links work.

The URL structure has also changed from `https://vote.usa.gov/states/<STATE_NAME>/` to `https://vote.usa.gov/register/<STATE_NAME>/` since some were territories.

All of the supported languages for Voter Registration forms are also [added in from the EAC website in this commit](https://github.com/18F/vote-gov/pull/63/commits/8f6613530df2af013f35a0fdb151f5e707186373).

This work tackles the issue #61

cc: @theresaanna

### CODE REVIEW

File: layouts/register/register-online.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   </header>
   <p class="usa-font-lead">{{ title .Title }} allows residents to register online.</p>
   <p>
-    <a href="{{ .Params.external_link }}">Start your online registration</a>
+    <a href="{{ .Params.external_link }}?ref={{ .Site.Params.Referrer_Short_Code }}">Start your online registration</a>
```
Comment (maya) [inline layouts/register/register-online.html]: @rogeruiz is there a reason we're not using the referrer code in `register-in-person.html` external links?

**Comment by maya** [comment]
@rogeruiz the "Learn more about voting” link on the “No registration needed” pages need external links. They are specific to each state's voting website, so we'll need to lift these from https://vote.usa.gov/ as well.

File: layouts/register/register-online.html (hunk 1 above)
Comment (rogeruiz) [inline layouts/register/register-online.html]: I took the referrer code examples from what's on `vote.usa.gov` currently, but we can definitely add it in another PR. I'll create an issue around it today.

**Comment by rogeruiz** [comment]
Ah the actual template @maya? I will update this PR.

**Comment by maya** [comment]
@rogeruiz yep, but it turns out it's just one! North Dakota ;)

**Comment by maya** [comment]
Passing on adding additional referral codes for now!

**Pull request merged**